travel/views.py

```diff
@@ -53,17 +53,20 @@ class TripList(LoginRequiredView, FilterMixin, ListView):
 
     def get_queryset(self, *args, **kwargs):
         qs = super(TripList, self).get_queryset(*args, **kwargs)
+        context_object_name = 'trip_list_in_order'
         return qs
+    queryset = Trip.objects.order_by("start_date")
```

Comment:
This sorts in ascending order but the requirement is for descending. Use a `-` before the property name to get descending. But was there a way to incorporate this into the `get_queryset` method instead of overwriting it? An alternative is simply a 
```    
class Meta:
        ordering = ["-start_date"]
```
on the Trip model without touching the view.